### ISSUE TITLE
Support for zarr zipstore

### ISSUE THREAD

**Issue by iprapas** [SeasFire/FireSight#32]
In huggingface you can upload zarrs as zipstores and then stream them and read in chunks.

It could be interesting to support zarr zipstores, as this would allow to just upload the dataset on the web and then just stream in firesight 

check this https://github.com/huggingface/datasets/issues/4096

**Comment by lazarusA** [comment]
yes, I have a branch where I did some experiments, still not working.
